html/cross-origin-embedder-policy/header-parsing.https.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <meta charset="utf-8">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+<body>
+<script>
+'use strict';
+function createIframe(t, values) {
+  const parent = document.createElement('iframe');
+  const child = document.createElement('iframe');
+  const params = new URLSearchParams(values.map((value) => ['value', value]));
+  parent.setAttribute('src', `resources/empty-coep.py?${params}`);
+  document.body.appendChild(parent);
+  t.add_cleanup(() => parent.remove());
+
+  return new Promise((resolve, reject) => {
+      parent.onload = resolve;
+      parent.onerror = reject;
+    })
+    .then(() => {
+        child.setAttribute('src', '/common/blank.html');
+        parent.contentDocument.body.appendChild(child);
+        return new Promise((resolve) => child.onload = resolve);
+      })
+    .then(() => child);
+}
+
+[
+  [],
+  [''],
+  ['jibberish'],
+  ['require\u2014corp'], // non-ASCII character (em dash)
+  ['require-corp;'],
+  ['require-corp\0'],
+  ['Require-corp'],
+  ['require-corp; foo=bar'],
```

Comment:
This doesn't seem right. Parameters are allowed per https://httpwg.org/http-extensions/draft-ietf-httpbis-header-structure.html#parse-item

Reply:
[The COEP proposal currently specifies that the value must be parsed as an `sh-token`.](https://mikewest.github.io/corpp/#COEP) That's also why the string and byte sequence values should be ignored.

Reply:
Those are authoring requirements. The processing model is given in https://mikewest.github.io/corpp/#abstract-opdef-obtain-a-responses-embedder-policy step 3.2.

Reply:
The text in the header's definition contains instructions for UAs, so this is a bit ambiguous. @domenic and I are following up in the proposal's repository: https://github.com/mikewest/corpp/pull/5

Reply:
In https://github.com/mikewest/corpp/pull/2 I required that any parameters present must fail parsing. But maybe parameters should be ignored instead? (Let's continue this thread in that PR.)